packages/@react-stately/selection/src/useMultipleSelectionState.ts

```diff
@@ -29,6 +29,8 @@ export function useMultipleSelectionState(props: MultipleSelection): MultipleSel
   // But we also need to trigger a react re-render. So, we have both a ref (sync) and state (async).
   let isFocusedRef = useRef(false);
   let [, setFocused] = useState(false);
+  let isFocusWithinCellRef = useRef(false);
+  let [, setFocusWithinCell] = useState(false);
```

Comment:
Can we get away with only the ref in this case? Do we need to trigger a re-render when this property updates?